docs/api-ref.md

```diff
@@ -4700,21 +4700,221 @@ See [ViewItem class](#viewitem-class)
 
 
 ---
+## Webhooks
+<br>
+<br>
+Using the Tableau Server Client (TSC), you can create a new webhook, get a list of all the Webhooks, get details about a specific webhook, or delete a webhook. 
+<br>
 
+The Webhook resource for Tableau Server and Tableau Cloud are defined in the WebhookItem class. The class corresponds to the Webhook resources you can access using the Tableau REST API.  For example, using REST API, you can gather information about a workbook, like the name of the Webbook, the event it is associated with, and the destination URL, and you can get the same information using TSC as well.
+<br>
 
-## Workbooks
+Tableau Webhook REST API endpoints are available in **REST API version 3.16** and later. 
+<br>
+<br>
 
-Using the TSC library, you can get information about a specific workbook or all the workbooks on a site, and  you can publish, update, or delete workbooks.
+### WebhookItem class
+The `WebhookItem` represents the Webhook resources on Tableau Server or Tableau Cloud. This is the information that can be sent or returned in response to a REST API request for Webhooks.  
 
-The project resources for Tableau are defined in the `WorkbookItem` class. The class corresponds to the workbook resources you can access using the Tableau REST API. The workbook methods are based upon the endpoints for projects in the REST API and operate on the `WorkbookItem` class.
+**Attributes**
 
+Name | Description
+:--- | :---
+`id` |  The identifier (*luid*) for the webhook. You need this value to query a specific Webhook  with the `get_by_id` method or to delete a Webhook with the  `delete` method.
+`name`  |  The name of the webhook. You must specify this when you create an instance of the `WebhookItem`.
+`url` | The destination URL for the webhook. The webhook destination URL must be https and have a valid certificate. You must specify this when you create an instance of the `WebhookItem`.
+`event` | The name of the Tableau event that triggers your webhook.This  is either `api-event-name` or `webhook-source-api-event-name`: one of this is required to create an instance of the `WebhookItem`.  <br> The event name must be one of the supported events listed in the Trigger Events table. The event and webhook-source use different name values for the same event.
+`owner_id` |  The identifier of the owner of the webhook.
 
+**Example**
 
+```py
+    import tableauserverclient as TSC
 
+    # Create new Webhook_item
+
+    new_webhook = TSC.WebhookItem()
+```
```

Comment:
For code sample like this which are "fenced" by the 3 backticks, you don't need the additional indenting. I.e. just slide these code rows all the way to the left, then it'll look consistent.